.travis.yml

```diff
@@ -4,13 +4,12 @@ cache:
   directories:
     - $HOME/.composer/cache
     - vendor
+    - $HOME/cphalcon
 php:
 - 5.3
 - 5.4
 - 5.5
 - 5.6
-- 7
```

Comment:
Don't remove php 7 and hhvm!! Just add an if statement around your before_install:

```yaml
  - composer install --prefer-source --no-interaction
  - if [[ ${TRAVIS_PHP_VERSION:0:1} == "5" ]]; then vendor/bin/install-phalcon.sh 2.0.x; fi;
  - phpenv config-rm xdebug.ini || return 0
```

.. Or something similar